public/admin/config.yml

```diff
@@ -673,5 +673,18 @@ collections:
         widget: "string"
         required: false
         hint: "Luma Calendar URL for the event. Include relevant tags when appropriate, e.g., https://lu.ma/filecoin-events?tag=fil%20bangkok%202024"
+      - name: luma-events-section
+        label: "Luma Events Section"
+        widget: "object"
+        required: false
+        fields:
+          - name: "title"
```

Comment:
Maybe we could add a hint about the type of the title? Provide a title for this Luma events section, such as 'Upcoming Events' or 'Keynote Sessions'?